### PR TITLE
Prototype version of Resonance Package

### PR DESCRIPTION
**Pull request by BongHwi**
`LFResonanceInitializer` will produce the dedicated table for RSN analysis.

`phianalysis.cxx` is an example of this task.

### CODE REVIEW

**Comment by njacazio** [comment]
Great work, we can also try the new merging !

**Review by njacazio — DISMISSED** [review]

File: PWGLF/DataModel/LFResonanceTables.h [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+DECLARE_SOA_COLUMN(Eta, eta, float);                                   //! Eta
+DECLARE_SOA_COLUMN(Phi, phi, float);                                   //! Phi
+DECLARE_SOA_COLUMN(PartType, partType, uint8_t);                       //! Type of the particle, according to resodaughter::ParticleType
+DECLARE_SOA_COLUMN(TempFitVar, tempFitVar, float);                     //! Observable for the template fitting (Track: DCA_xy, V0: CPA)
+DECLARE_SOA_COLUMN(Indices, indices, int[2]);                          //! Field for the track indices to remove auto-correlations
+DECLARE_SOA_COLUMN(Sign, sign, int8_t);                                //! Sign of the track charge
+DECLARE_SOA_COLUMN(TPCNClsCrossedRows, tpcNClsCrossedRows, uint8_t);   //! Number of TPC crossed rows
+DECLARE_SOA_COLUMN(DcaXY, dcaXY, float);                               //! DCA_xy
+DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);                                 //! DCA_z
+DECLARE_SOA_COLUMN(X, x, float);                                       //! x position of the track
```
Comment (njacazio) [inline PWGLF/DataModel/LFResonanceTables.h:90]: This is a redefinition of https://github.com/AliceO2Group/AliceO2/blob/dev/Framework/Core/include/Framework/AnalysisDataModel.h#L98 you could consider avoid this duplication and use directly `aod::X` in your table

File: PWGLF/DataModel/LFResonanceTables.h [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                  resodaughter::Eta,
+                  resodaughter::Phi,
+                  resodaughter::PartType,
+                  resodaughter::TempFitVar,
+                  resodaughter::Indices,
+                  resodaughter::Sign,
+                  resodaughter::TPCNClsCrossedRows,
+                  resodaughter::DcaXY,
+                  resodaughter::DcaZ,
+                  resodaughter::X,
```
Comment (njacazio) [inline PWGLF/DataModel/LFResonanceTables.h:124]: Here you could use `aod::X`

File: PWGLF/DataModel/LFResonanceTables.h (hunk 1 above)
Comment (BongHwi) [reply PWGLF/DataModel/LFResonanceTables.h:90]: That's a reasonable suggestion. I'll update the table.

File: PWGLF/DataModel/LFResonanceTables.h (hunk 2 above)
Comment (BongHwi) [reply PWGLF/DataModel/LFResonanceTables.h:124]: That's new for me. Thanks for letting me know!

**Review by njacazio — APPROVED** [review]
The new utility for the event cuts is super nice!

**Pull request merged**